Compiler/Compiletime/syntax_check.py

```diff
@@ -196,3 +196,19 @@ def check_ElseStmt(tokens):
         valid = False
         reason = 'Missing colon at the end in the used else stmt.'
     return [valid, reason]
+
+def check_ClassStmt(tokens):
+    '''
+    This function will be checking the impl of the syntax of class
+    '''
+    valid = True
+    reason = ''
+    if tokens[len(tokens)-1].token != ":":
+        valid = False
+        reason = 'Missing colon at the end in the used class stmt.'
+    else:
```

Comment:
avoid the pass keyword and the else block if there are no statements inside  there is no need for the else in this context.